src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation/navigation.component.scss

```
@@ -24,6 +24,14 @@
   height: 25px;
 }
 
+cds-header-item {
+  width: 600px;
+}
+
+.cluster-switcher {
+  margin-left: 8rem;
```

6 rem looks better from alignment perspective.
```suggestion
  margin-left: 6rem;
```